http/src/test/resources/reference.conf

```diff
@@ -267,3 +266,5 @@ akka.ssl-config {
     ]
   }
 }
+
+opencensus-scala.trace.sampling-probability = 1.0
```

Comment:
Do we need to set this in the `main` reference.conf? What is the probability currently set to?

Reply:
yea..there's a firecloud PR https://github.com/broadinstitute/firecloud-develop/pull/2130 ...the PR is still set to 1 for testing....I'm thinking setting it to 0.5, what do you think?

Reply:
Cool, yeah 0.5 or even 0.1 seems reasonable

Reply:
Oh just want to document here, the reason why `opencensus-scala.trace.sampling-probability = 1.0` needs to live in `firecloud-develop` is because `opencensus-scala` defines a bunch config in the library and `reference.conf` won't be able to overwrite it...will add a comment in broadinstitute/firecloud-develop#2130